src/api.jl

```diff
@@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
     # Get indicies of entries in the  RA/Dec range of interest.
     entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
```

Comment:
If you do concatenate all the catalogs at the beginning, then that would, I think, answer my objection.  Each object gets matched to tiles in a particular image using its catalog entry and the PSF from the center of the image.  I'd be reassured by a unit test demonstrating that this works as described, though.